instruments/keithley/keithley580.py

```diff
@@ -342,7 +342,7 @@ def get_status_word(self):
             time.sleep(1)
             statusword = self.query('')
 
-        if statusword is None:
```

Comment:
The `statusword` can never be `None` in this scenario. The `while` loop above will, if unsucessfull, leave it as an empty string. If something wrong comes back from the instrument, this string will not be empty, but invalid (which the while loop checks for). So if after 5 tries nothing valid comes back, the counter will be at zero. Thus let's check for `tries == 0`.